gluten-core/src/main/scala/org/apache/spark/util/GlutenShutdownManager.scala

```diff
@@ -18,12 +18,18 @@ package org.apache.spark.util
 
 object GlutenShutdownManager {
 
+  private def isJava8: Boolean = System.getProperty("java.version").equals("8")
```

Comment:
@lwz9103 colud you please check whether we set this environment variable our CI environment?

Reply:
This is a property exposed by the JVM, it should be present in all the JVM

Reply:
But I think before Java 9, the java.version is set to 1.major.minor_subminor. I'll change this to parse the major version.